account_financial_report_qweb/wizard/trial_balance_wizard.py

```diff
@@ -131,10 +156,10 @@ def onchange_show_partner_details(self):
         """Handle partners change."""
         if self.show_partner_details:
             self.receivable_accounts_only = self.payable_accounts_only = True
-            self.hide_account_balance_at_0 = True
+            self.hide_account_at_0 = True
```

Comment:
Now that hide_account_at_0  is True by default, it should be completely removed from this onchange method onchange_show_partner_details() (2 lines of code to remove). Otherwise, the default value at True is immediately removed by this onchange !